examples/generalPermissionManager.ts

```diff
@@ -129,27 +128,35 @@ window.addEventListener('load', async () => {
   await generalPM.addDelegate({ delegate: myAddress, details: 'details' });
 
   const generalTMAddress = (await tickerSecurityTokenInstance.getModulesByName({
-    moduleName: ModuleName.generalTransferManager,
+    moduleName: ModuleName.GeneralTransferManager,
   }))[0];
 
   // Get all delegates
-  await generalPM.changePermission({ valid: true, perm: 'FLAGS', delegate: myAddress, module: generalTMAddress });
+  await generalPM.changePermission({ valid: true, perm: 'ADMIN', delegate: myAddress, module: generalTMAddress });
 
   // Check  delegate
   console.log('Delegate is added:');
   console.log(await generalPM.checkDelegate({ delegate: myAddress }));
   console.log('Delegate has flags perm added on general transfer manager:');
   console.log(await generalPM.checkPermission({ delegate: myAddress, module: generalTMAddress, permission: 'FLAGS' }));
```

Comment:
These console.log statements should change to reflect the change in permission used